classes/user/UserStageAssignmentDAO.inc.php

```diff
@@ -104,7 +104,7 @@ function filterUsersNotAssignedToStageInUserGroup($submissionId, $stageId, $user
 			(int) $userGroupId,
 		];
 		if ($name !== null) {
-			$params = array_merge($params, array_fill(0, 6, '%'.(string) $name.'%'));
+			$params = array_merge($params, array_fill(0, 6, '%'.strtolower((string) $name).'%'));
```

Comment:
Doing the `strtolower` of the parameter in PHP and then matching with `LOWER(column) LIKE ?` in SQL invites tricky collation problems. I would suggest doing the case conversion 100% on the SQL side by using something like `LOWER(column) LIKE CONCAT('%', ?, '%')`.

Reply:
See this pattern: https://github.com/pkp/ojs/blob/858656d68a24bba35238667b0eda9d528a0e5d8e/classes/issue/Collector.php#L436